products/postfix.md

```diff
@@ -4,14 +4,16 @@ category: server-app
 permalink: /postfix
 versionCommand: postconf mail_version
 releasePolicyLink: https://www.postfix.org/announcements.html
-changelogTemplate: https://www.postfix.org/announcements/postfix-__LATEST__.html
+changelogTemplate: https://www.postfix.org/announcements/postfix-3.8.1.html
```

Comment:
I think it would be better if we drop the `changelogTemplate` entirely.

And maybe we should only link only the 'x.y.0' release announcements for non-EOLed versions in order to lower the maintenance burden ?

Reply:
Dropping changelogTemplate. 6 updates a year or so sounds doable to me. imo,links should be consistent throughout the page - all to x.y.0, or all to the latest.

Now watching the page as well, will keep track of updating latest links.

Reply:
Dropped changelogTemplate